libenkf/src/queue_config.c

```diff
@@ -223,6 +223,11 @@ bool queue_config_init(queue_config_type * queue_config, const config_content_ty
   if (config_content_has_item(config_content, JOB_SCRIPT_KEY))
     queue_config_set_job_script(queue_config, config_content_get_value_as_abspath(config_content, JOB_SCRIPT_KEY));
 
+  if (config_content_has_item(config_content, MAX_SUBMIT_KEY)) {
+    queue_config->max_submit = config_content_get_value_as_int(config_content, MAX_SUBMIT_KEY);
+    queue_config->max_submit_set = true;
```

Comment:
Not entirely sure. This is legacy code from site_config, where max_submit_set (and max_submit) was duplicated and lived in both site_config and queue_config. I removed the one in site_config and simultaneously moved the loading of these variables into queue_config.